css/selectors.json

```diff
@@ -272,6 +272,14 @@
     ],
     "status": "standard"
   },
+  ":matches()": {
+    "syntax": ":matches( selector[, selector]* )",
```

Comment:
`selector[, selector]*` -> `<selector>#`
Actually it takes a `<selector-list>` (https://www.w3.org/TR/selectors-4/#matches), but this syntax is not defined yet. Grammar can be taken here: https://www.w3.org/TR/selectors-4/#grammar